db/core/ecmcAxis.db

```diff
@@ -506,3 +506,66 @@ record(longin,"${P}${AXIS_NAME}-ErrId"){
   field(HIGH, 1)
   field(HSV,  "MAJOR")
 }
+
+record(mbbo,    "${P}${AXIS_NAME}-movVelCmd") {
+  field(DESC, "absolute move command data")
+  field(VAL,  "${VELPROC=0}")
+
+  field(ZRVL, 0)  field(ZRST, "normal")
+
+  field(FLNK, "${P}${AXIS_NAME}-MtnCmd_.PROC")
+}
+
+record(mbbo,    "${P}${AXIS_NAME}-movRelCmd") {
+  field(DESC, "absolute move command data")
+  field(VAL,  "${RELPROC=0}")
+
+  field(ZRVL, 0)  field(ZRST, "normal")
+
+  field(FLNK, "${P}${AXIS_NAME}-MtnCmd_.PROC")
+}
+
+record(mbbo,    "${P}${AXIS_NAME}-movAbsCmd") {
+  field(DESC, "absolute move command data")
+  field(VAL,  "${ABSPROC=0}")
+
+  field(ZRVL, 0)  field(ZRST, "normal")
+  field(ONVL, 1)  field(ONST, "gear in")
```

Comment:
"gear in" is probably better written as "gearin" (or GEARIN) ?
And then "normal" could become GEAROUT ?

Reply:
What is wrong with a space in the string? Is there a convention I'm missing? I noticed that many/all enum strings are like that. Any reason for this?
It's two words, and if no spaces are allowed, than it should be 'gearIn' or 'GearIn'. The PLCopenMotion function close to this move type is `MC_GearIn`.
As two normal being gear out, no. Normal is a standard absolute move.
For details, please talk to @anderssandstrom